src/renderer/components/common/SignatoryCard/SignatoryCard.tsx

```diff
@@ -26,13 +27,21 @@ const SignatoryCard = ({
   addressFont = 'text-body text-inherit',
   size = 20,
   name,
+  matrixId,
   ...addressProps
 }: Props) => {
   const { getLiveAccounts } = useAccount();
   const address = getAddress(addressProps);
   const account = getLiveAccounts().find((a) => toAddress(a.accountId) === address);
```

Comment:
In worst case we're guaranteed to receive `address`
`name={getSignatoryName(accountId, tx, contacts, accounts, connection?.addressPrefix)}`
So condition will never fall into `account?.name` - `<AccountAddress ... name={name || account?.name} />`
Guess it's safe to remove `useAccount` from `SignatoryCard`
Please correct me if I'm wrong 🙏